### PR TITLE
[C345410] Search mapping profiles (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by AliaksandrShuleika**
[testrail](https://foliotest.testrail.io/index.php?/cases/view/345410)
[report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4875/allure/#suites/269d518d2951593dccd78f4d0215297f/c490889415db597d/)

### CODE REVIEW

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js [hunk 1]
```diff
@@ -74,6 +77,37 @@ export default {
       });
   },
 
+  searchFieldMappingProfile(text) {
+    cy.do(searchField.fillIn(text));
+  },
+
+  verifySearchButtonEnabled: () => cy.expect(searchButton.has({ disabled: false })),
+  verifySearchButtonDisabled: () => cy.expect(searchButton.has({ disabled: true })),
```
Comment (nayimovag) [inline cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js:85]: merge these into one method where a true or false argument is passed, like in the following example:
  isUsersRadioChecked(checked = true) {
    cy.expect(usersRadio.has({ checked }));
  },

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js [hunk 2]
```diff
@@ -74,6 +77,37 @@ export default {
...
+
+  verifyClearSearchButtonExists: () => cy.expect(clearButton.exists()),
+  verifyClearSearchButtonAbsent: () => cy.expect(clearButton.absent()),
+
+  clearSearchField: () => {
+    cy.do(searchField.focus());
+    cy.do(clearButton.click());
```
Comment (nayimovag) [inline cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js:92]: use array, like this 
    cy.do([
      searchField.focus(),
      clearButton.click()
    ]);

**Review by nayimovag — CHANGES_REQUESTED** [review]

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js (hunk 1 above)
Comment (AliaksandrShuleika) [reply cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js]: fixed

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js (hunk 2 above)
Comment (AliaksandrShuleika) [reply cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js]: fixed

**Review by dnovikau — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

**Pull request merged**